compiler+runtime/doc/install.md

```diff
@@ -66,6 +66,33 @@ yay -S jank-git   # if you installed jank-git
 yay -S jank-bin   # if you installed jank-bin
 ```
 
+## Nix
+### Install
+We have binary packages in our cachix cache, so installation is quick and easy.
+You may also skip the cache setup and artifacts will be built from source.
+
+```bash
+# Make sure cachix is installed, then follow the prompts.
+cachix use jank-lang
+```
+
+Next, you can directly run the jank binary from the nix flake derivation. For
+example, to run the `jank check-health` subcommand:
+
+```bash
+nix run git+https://github.com/jank-lang/jank.git -- check-health
```

Comment:
Does this provide us with any means to install jank properly, such that there's a `jank` binary available within the user's shell? Or is that a later step, when we get into nixpkgs?